.gitignore

```diff
@@ -16,4 +16,4 @@ yarn.lock
 /tests/e2e/screenshots
 /tests/e2e/videos
 .lighthouseci
-lighthouseReport
\ No newline at end of file
+lighthouseReport
```

Comment:
nitpick: omit this change? there's no change

Reply:
erm but should have a newline at the end of file right?

Reply:
should be handled by prettier, may want to check if it's default rules are been enforced or not

Reply:
yup it does enforce it, but it does not enforce in this file. so i manually add it in. is that not ok? if so i can remove it.

Reply:
oh, then leave it lo. probably we should bake the check in pipeline then next time